.azure/gpu-tests-fabric.yml

```diff
@@ -96,13 +105,21 @@ jobs:
         for fpath in `ls requirements/**/*.txt`; do \
           python ./adjust-torch-versions.py $fpath ${PYTORCH_VERSION}; \
         done
+      condition: eq(variables.IS_NIGHTLY, 'false')
```

Comment:
This shall be still covered in the script

Reply:
What do you mean?

Reply:
@Borda Could you make a suggestion for a follow-up. It would be nice if we could unblock ourselves for testing with nightly :))